pdk/1.0/modules/ROOT/pages/policies-pdk-debug-local.adoc

```diff
@@ -1,14 +1,14 @@
-= Debugging Custom Policies Locally with PDK
+= Debugging Custom Policies with the PDK Debugging Playground
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-To test the functionality of your policy, Policy Development Kit (PDK) provides a pre-configured API instance located in the `<root-directory>/playground/config` directory.
+To test the functionality of your policy, Flex Gateway Policy Development Kit (PDK) provides a pre-configured API instance located in the `<root-directory>/playground/config` directory.
 
-After you successfully xref:policies-pdk-compile-policies.adoc[compile your custom policy project] and add a <<register, Flex Gateway registration file>> to the configuration directory, you can use the `run` command to deploy the API instance and your policy to begin testing.
+The PDK Debugging Playground only supports Flex Gateways running in Local Mode in a Docker container. If you plan to deploy or policy to a Flex Gateway running in Connected Mode or to a different platform, you should still test your policy in PDK by using Flex Gateways running in Local Mode in a Docker container for initial testing.
```

Comment:
```suggestion
The PDK Debugging Playground only supports Flex Gateway running in Local Mode in a Docker container. If you plan to deploy a policy to Flex Gateway running in Connected Mode or to a different platform, you should still first test your policy in PDK by using Flex Gateway running in Local Mode in a Docker container.
```
I'm not sure what "or to a different platform" is referring to. You first mention "Connected Mode" - are you referring to its alternative: Local Mode? Or is this referring to something else? Production environment perhaps?